### ISSUE TITLE
NVDA is announcing as "blank" before every list item once the user navigates to data validation list.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#16238]
### Steps to reproduce:
Step 1: Turn on NVDA Screen reader and open Excel on web.
Step 2: Excel Workbook is opened, now focus is on grid cell.
Step 3: Navigate to ribbon using Alt+F6 key.
Step 4: Navigate to Data tab using arrow key and activate it with enter key.
Step 5: Navigate to Data Validation tab using arrow key and activate it. Select the list from the settings tab and give some of the list data.
Step 6: Now navigate to the selected cells where data validation is applied.
Step 7: Expand the dropdown using Alt+ Down arrow key.
Step 8: Navigate in list.
Step 9: Verify if "blank" word is announced before every list item or not.

### Actual behavior:
NVDA is announcing as "blank" before every list item once the user navigates to data validation list.

### Expected behavior:
Unnecessarily "blank" word should not announce with list item once the user navigates to data validation list using NVDA screen reader. 

### NVDA logs, crash dumps and other attachments:
[Issue Repro.webm](https://github.com/nvaccess/nvda/assets/93735775/3dd9b3d1-6e1c-4d0f-b444-ec82322a06f4)

### System configuration:
Excel Version: 16.0.16816.42307
Browser details: Microsoft Edge Version 115.0.1901.203 (Official build) (64-bit)
Edition Windows 11 Enterprise Insider Preview
Version 23H2
OS build 25905.1000.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2023.3.3 (2023.3.3.30854)

#### Windows version:
OS build 25905.1000.

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Previous version they have added

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

To be able to reproduce and investigate this issue, we need a **MINIMAL** code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

Kind Regards,
NV Access Software Developers
